.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
```

Comment:
Given these are very unlikely to fail they should run on push only. But to manifest they are working there should be a separate commit marked with `drop!` to run them on pull request in this PR.

Reply:
oh, yeah, I've only got that now to test out the containerized build (the build container needs some work to build openssl from the target branch, rather than having it be built once when the container is built)

Reply:
@nhorman do you plan to change triggers to `push` only as @t8m proposed?

Reply:
oh, yeah, forgot to reset that after I finished my testing, will fix

Reply:
fixed